### ISSUE TITLE
DummyClassifer

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#18]
scikit-learn has a [DummyClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html).  Adding one of these could be a good way to teach about benchmarking against naive baselines and understanding how performance metrics depend on class label distributions.
